### PR TITLE
Added lumin component installers

### LINKED ISSUE #147: Add Lumin component install support
Add the ability to download the Lumin components

### PR DESCRIPTION
**Pull request by StephenHodgson**
Note, this may not be availible until 2019 gets into beta or until we can download/install alpha builds, but I went ahead and did the preliminary work.

Fixes #147

### CODE REVIEW

**Comment by StephenHodgson** [comment]
Agreed, just got started on the work for early feedback.

**Comment by StephenHodgson** [comment]
@jwittner 2019 beta is out with accessible download components

**Comment by StephenHodgson** [comment]
This PR is ready for review

**Comment by jwittner** [comment]
Note: this will conflict with #161 .

**Comment by jwittner** [comment]
I'll pull this down and test it out tomorrow. =)

**Comment by jwittner** [comment]
Bah, UnitySetup throws an error if I search for 2019.1.0b9

**Comment by StephenHodgson** [comment]
This should be ready to test once the 2019 bugs are fixed.

File: UnitySetup/UnitySetup.psm1 [hunk 1]
```diff
@@ -1310,7 +1313,7 @@ function Start-UnityEditor {
         [parameter(Mandatory = $false)]
         [string]$LogFile,
         [parameter(Mandatory = $false)]
-        [ValidateSet('StandaloneOSX', 'StandaloneWindows', 'iOS', 'Android', 'StandaloneLinux', 'StandaloneWindows64', 'WebGL', 'WSAPlayer', 'StandaloneLinux64', 'StandaloneLinuxUniversal', 'Tizen', 'PSP2', 'PS4', 'XBoxOne', 'N3DS', 'WiiU', 'tvOS', 'Switch')]
+        [ValidateSet('StandaloneOSX', 'StandaloneWindows', 'iOS', 'Android', 'StandaloneLinux', 'StandaloneWindows64', 'WebGL', 'WSAPlayer', 'StandaloneLinux64', 'StandaloneLinuxUniversal', 'Tizen', 'PSP2', 'PS4', 'XBoxOne', 'N3DS', 'WiiU', 'tvOS', 'Switch', 'Lumin')]
```
Comment (jwittner) [inline UnitySetup/UnitySetup.psm1]: I haven't tried to build for Lumin, but I don't see this as a build target in either of these documentation pages:

https://docs.unity3d.com/Manual/CommandLineArguments.html
https://docs.unity3d.com/ScriptReference/BuildTarget.html

I'm wondering if you've verified the effects of calling `Start-UnityEditor -BuildTarget Lumin`.

Comment (StephenHodgson) [reply UnitySetup/UnitySetup.psm1]: Unknown. I need to test this out still 
https://forum.unity.com/threads/magic-leap-command-line-usage.560836/#post-4451941

Comment (jwittner) [reply UnitySetup/UnitySetup.psm1]: Ok. Give it a shot, if it doesn't work, lets drop it from this list until it's supported by Unity. =)

Comment (StephenHodgson) [reply UnitySetup/UnitySetup.psm1]: Sounds good :)

**Comment by StephenHodgson** [comment]
@jwittner, I was able to validate that the command like works! :shipit:

**Pull request merged**